### PR TITLE
Create-and-remove-books

### PR DESCRIPTION
**Pull request by Donard97**
**I created this PR to check if our project was as requested.**

_This is a Bookstore project when you can add a book, and remove selected books. In this milestone, I had to write actions and reducer to add and remove books._

### Project Requirement:

> - Imported createStore ✔️ 
> - Added functions to add and remove books ✔️  
> - Writed a reducer for books ✔️ 
> - Configured configureStore file ✔️ 

<hr>

_Feel free to review my project and let me know if there are any essential changes 💪 ._

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Donard97, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js [hunk 1]
```diff
@@ -1,11 +1,59 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (input.title.trim() && input.author.trim()) {
+      props.propsToAddBooks(input.title, input.author);
+      setInput({
+        title: '',
+        author: '',
+      });
+    }
+  };
```
Comment (mke2111) [inline src/components/AddBook.js]: - [x] For this milestone, you are required to generate a unique ID while saving the book data from the inputs, and then dispatches the addBook action with the object as its argument. All this should be in this file.

Comment (Donard97) [reply src/components/AddBook.js]: Thank you for letting me know. I moved everything that you asked for on the AddBook file. Here is a screenshot of the function:   
image

**Review by Dipeshtwis — APPROVED** [review]
# Hi @Donard97, 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It is better if you put `.env` in the `.gitignore` because most of the time `.env` contains the secret key. So be aware while doing that.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**